pullrequest.rst

```diff
@@ -114,6 +108,21 @@ You should have already :ref:`set up your system <setup>`,
    git commit -m '<message>'
    git push origin <branch-name>
 
+  * If someone else added new changesets and you get an error::
+
+     git checkout <branch-name>
+     git pull origin <branch-name>  # pull = fetch + merge
+     # if there are merge conflicts: see "Resolving Merge Conflicts" below
+     git push origin <branch-name>
```

Comment:
I think this part is unclear.

* Where someone else added new changesets to?
* What type of error you got?

Reply:
Moving this block from where is was is an improvement, but I think the following is clearer and fits into the new context even better.
```
* If a core developer reviewing your PR pushed a commit to your PR branch, then add

  git pull origin <branch-name>  # pull = fetch + merge

  after checking out your branch and before editing.  If you have made local changes that have not been pushed
  to your fork and there are merge conflicts, see "Resolving Merge Conflicts" below.
```

Reply:
Thanks, those are good comments! I've pushed a change, let me know if you think it's good.

Reply:
Good enough!